var/spack/repos/builtin/packages/dd4hep/package.py

```diff
@@ -152,6 +153,7 @@ class Dd4hep(CMakePackage):
     depends_on("lcio", when="+lcio")
     depends_on("edm4hep", when="+edm4hep")
     depends_on("podio", when="+edm4hep")
+    depends_on("podio@0.16:", when="@1.24:" +edm4hep")
```

Comment:
```suggestion
    depends_on("podio@0.16:", when="@1.24: +edm4hep")
```
I think this is a typo, otherwise looks good to me, thanks!

Reply:
Yes, typo, auto-quotes when I'm on my phone...